pkg/netpol/diff/diff.go

```diff
@@ -212,10 +237,10 @@ func (c *ConnsPair) updateNewOrLostFields(isFirst bool, peersList []string) {
 		src, dst = c.secondConn.Src(), c.secondConn.Dst()
 	}
 	// update src/dst status based on the peerList , ignore ips/ingress-controller pod
-	if !(src.IsPeerIPType() || src.IsFakePeer()) && !slices.Contains(peersList, src.String()) {
+	if !(src.IsPeerIPType() || isIngressControllerPeer(src)) && !slices.Contains(peersList, src.String()) {
```

Comment:
here we want to avoid using `slices.Contains`, and just look for `src.String` in a map (`map[string]bool`)

Reply:
done